.github/workflows/tests.yml

```diff
@@ -15,25 +15,25 @@ on:
     branches: master
   schedule:
     # * is a special character in YAML so you have to quote this string
-    - cron:  '0 3 * * 6'
+    - cron: "0 3 * * 6"
   workflow_dispatch:
     inputs:
       reason:
-        description: 'Reason'
+        description: "Reason"
         required: false
-        default: 'Manual trigger'
+        default: "Manual trigger"
 
 jobs:
   Tests:
     runs-on: ubuntu-20.04
     timeout-minutes: 20
     strategy:
       matrix:
-          python-version: [3.6, 3.7, 3.8, 3.9]
-          requirements-level: [min, pypi]
-          cache-service: [redis]
-          db-service: [postgresql9, postgresql11, mysql5, mysql8]
-          exclude:
+        python-version: [3.7, 3.8, 3.9]
+        requirements-level: [min, pypi]
+        cache-service: [redis]
+        db-service: [postgresql9, postgresql11, mysql5, mysql8]
+        exclude:
```

Comment:
```python
        exclude:
          - python-version: 3.7
            db-service: postgresql13

          - python-version: 3.7
            db-service: mysql8

        include:
          - db-service: postgresql10
            DB_EXTRAS: "postgresql"

          - db-service: postgresql13
            DB_EXTRAS: "postgresql"

          - db-service: mysql5
            DB_EXTRAS: "mysql"

          - db-service: mysql8
            DB_EXTRAS: "mysql"

          - search-service: elasticsearch7
            SEARCH_EXTRAS: "elasticsearch7"
```
based on https://github.com/inveniosoftware/invenio-records-resources/blob/master/.github/workflows/tests.yml

Reply:
I can't figure out what the change is here. GitHub doesn't show a button to merge changes, so I'm not sure what's going on. Can you clarify?